espmonitor/src/types.rs

```diff
@@ -81,11 +84,16 @@ impl Chip {
 
 impl Chip {
     pub fn target(&self, framework: Framework) -> String {
-        let mut target = String::from("xtensa-");
+        let mut target = String::new();
+        target.push_str(match self {
+            Chip::ESP32C3 => "riscv32imc-",
+            _ => "xtensa-",
+        });
         target.push_str(match self {
             Chip::ESP32 => "esp32-",
             Chip::ESP32S2 => "esp32s2-",
             Chip::ESP8266 => "esp8266-",
+            Chip::ESP32C3 => "unknown-",
```

Comment:
Hm, looks like the convention is a little different for the RISC-V variants.  For baremetal this is right, but for esp-idf it'll be `riscv32imac-esp-espidf`, but this will output `riscv32imac-unknown-espidf`.

Reply:
thanks, fixed